vsphere/image/settings.auto.pkrvars.hcl

```diff
@@ -0,0 +1,33 @@
+/*
+    DESCRIPTION:
+    Ubuntu Server 20.04 LTS variables used by the Packer Plugin for VMware vSphere (vsphere-iso).
+*/
+
+// Guest Operating System Metadata
+vm_guest_os_language = "en_US"
+vm_guest_os_keyboard = "us"
+vm_guest_os_timezone = "UTC"
+vm_guest_os_type = "ubuntu64Guest"
+
+
+// Virtual Machine Hardware Settings
+vm_firmware              = "efi"
+vm_cpu_count             = 2
+vm_mem_size              = 2048
+vm_disk_size             = 40960
+vm_disk_thin_provisioned = true
+vm_interface_name         = "vmxnet3"
```

Comment:
is the interface_name constant or depends on env ?

Reply:
Sorry, this misnamed variable actually represents the virtualized network card. vmxnet3 is the most recent paravirtualized network card and is default in my setup but I am not sure if it really is! I believe it's best to specify to make sure we are consistent. Maybe, I can rename this variable to vm_network_card ?

Reply:
You can add a brief explanation in the README